src/shared/pages/team/reactivateTeam/reactivateTeam.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import { ApplicationConsumer, ApplicationState } from '../../../contexts/application';
+import { getTeam, updateTeam } from '../../../services/teamsService';
+import ErrorMessage from '../../../models/errorMessage';
+import {
+    GENERAL_ERROR_TITLE,
+    LOAD_TEAM_ERROR_DESCRIPTION,
+    TEAM_UPDATE_FAILED_UNKNOWN_ERROR
+} from '../../../models/constants';
+import useError from '../../../hooks/useError';
+import ErrorSummary from '../../../common/components/errorSummary';
+
+
+interface MatchParams {
+    teamId: string;
+}
+
+interface ReactivateTeamProps extends RouteComponentProps<MatchParams> {
+    hasRole: (role: string) => boolean;
```

Comment:
You have the hasRole here but don't use it? Would this not also be useful to redirect if the user doesn't have permission to view the page?